Perpetuum.DataDumper/Views/ModuleERPDataView.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Perpetuum.Containers;
+using Perpetuum.Modules;
+using Perpetuum.Modules.Weapons;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Perpetuum.DataDumper.DataDumper;
+
+namespace Perpetuum.DataDumper.Views {
+    public class ModuleERPDataView : ActiveModuleDataView {
+
+        public double module_recovery_chemical { get; set; }
```

Comment:
I'll assume the casing here is because the variable name is used for something?
Otherwise stick to WhateverThisIsCalled

Reply:
I did it this way because it's what the examples on the mediawiki templates were using, and the casing here is reflected and used for the variable names. It seems common to use this in mediaWIki templates, but there's no reason we can't do something different.